ESP8266/ESP8266.cpp

```diff
@@ -215,7 +220,7 @@ void ESP8266::_packet_handler()
     uint32_t amount;
 
     // parse out the packet
-    if (!_parser.recv(",%d,%d:", &id, &amount)) {
+    if (!_parser.recv(",%d,%" SCNu32 ":" , &id, &amount)) {
```

Comment:
Old string throws out ugly warnings. I would still argue for a cleaner code and output.